packages/features/shell/Shell.tsx

```diff
@@ -340,24 +340,24 @@ function UserDropdown({ small }: UserDropdownProps) {
         <button
           className={classNames(
             "hover:bg-emphasis group mx-0 flex cursor-pointer appearance-none items-center rounded-full text-left outline-none focus:outline-none focus:ring-0 md:rounded-none lg:rounded",
-            small ? "p-2" : "px-2 py-1"
+            small ? "p-2" : "px-2 py-1.5"
           )}>
           <span
             className={classNames(
-              small ? "h-4 w-4" : "h-6 w-6 ltr:mr-2 rtl:ml-2",
+              small ? "h-4 w-4" : "h-5 w-5 ltr:mr-2 rtl:ml-2",
               "relative flex-shrink-0 rounded-full bg-gray-300"
             )}>
             <Avatar
-              size={small ? "xs" : "sm"}
+              size={small ? "xs" : "xsm"}
```

Comment:
Was this in intentional

Reply:
Yea I want the avatar to be 20x20px - so if that's indeed what this does - then yes!